samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/DocumentImportController.cs

```diff
@@ -86,12 +85,18 @@ private enum SupportedFileType
             return this.BadRequest("File size exceeds the limit.");
         }
 
-        if (documentImportForm.DocumentScope == DocumentImportForm.DocumentScopes.Chat
-            && !(await this.UserHasAccessToChatAsync(documentImportForm.UserId, documentImportForm.ChatId)))
+        var chatSession = await this._chatSessionRepository.FindByIdAsync(documentImportForm.ChatId.ToString());
+        if (chatSession == null)
         {
-            return this.BadRequest("User does not have access to the chat session.");
+            return this.NotFound("Session does not exist.");
         }
 
+        if (documentImportForm.DocumentScope == DocumentImportForm.DocumentScopes.Chat && authInfo.UserId != chatSession.UserId)
```

Comment:
This is still a problem because any user can upload a document for all users without any validation, but perhaps that was by design?

Reply:
Yes, this is by design